libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts

```diff
@@ -561,28 +561,22 @@ describe('Autocomplete component', () => {
     it('should handle missing skyAutocomplete directive on change', fakeAsync(() => {
       fixture.detectChanges();
 
-      try {
-        component.autocomplete['inputDirective'] = undefined;
+      component.hideInput = true;
+
+      expect(() => {
         tick();
         fixture.detectChanges();
-      } catch (e) {
-        expect(
-          e.message.indexOf(
-            'The SkyAutocompleteComponent requires a ContentChild input'
-          ) > -1
-        ).toEqual(true);
-      }
+      }).toThrowError(
+        'The SkyAutocompleteComponent requires a ContentChild input or textarea bound with the SkyAutocomplete directive. ' +
+          'For example: `<input type="text" skyAutocomplete>`.'
+      );
     }));
 
     it('should set the width of the dropdown when a search is performed', fakeAsync(() => {
       const adapterSpy = spyOn(
         autocomplete['adapterService'],
```

Comment:
This seems like a place where we should either use `TestBed.overrideComponent()` to provide our own adapter service that we spy on, or use `fixture.debugElement.injector.get()` to get the adapter service, instead of accessing a private member.

Reply:
Good call. Done.